### PR TITLE
Update dehelper from 4.3.1 to 4.3.3

### PR DESCRIPTION
**Pull request by TsanBoi**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Review by p-linnane — DISMISSED** [review]

File: Casks/dehelper.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask "dehelper" do
-  version "4.3.1"
+  version "4.3.2"
```
Comment (razvanazamfirei) [inline Casks/dehelper.rb:2]: What's the source of this version? Does the livecheck need to be updated?

Comment (Vulxan) [reply Casks/dehelper.rb:2]: The source is from the livecheck url, and the result of running livecheck from my terminal is:
```
❯ brew livecheck dehelper
dehelper: 4.3.1 ==> 4.3.2
```

Comment (razvanazamfirei) [reply Casks/dehelper.rb:2]: Gotcha, it looks like it hasn't propagated everywhere.

Comment (Vulxan) [reply Casks/dehelper.rb:2]: Should the PR be closed for this situation?😂

Comment (razvanazamfirei) [reply Casks/dehelper.rb:2]: I was able to get the 4.3.2 version. Let's give it more time.

**Review by p-linnane — DISMISSED** [review]

**Comment by p-linnane** [comment]
The CI will keep rerunning periodically now and merge when it returns the new version.

**Review by razvanazamfirei — APPROVED** [review]

**Pull request merged**